### PR TITLE
Don't mention the Symfony Standard edition in the contributing docs

### LINKED ISSUE #11692: [Community Reviews] outdated advice to use Symfony Standard
In the [Community Reviews doc](https://github.com/symfony/symfony-docs/blob/cc9d8ece0d582831be3e7edc9e2c14141d34a879/contributing/community/reviews.rst) we are advised to reproduce problems in the Symfony Standard edition, but the [README](https://github.com/symfony/symfony-standard/blob/3.4/README.md) for the Symfony Standard edition contains a warning which seems to indicate that this advice is outdated:-

> WARNING: This distribution does not support Symfony 4. See the Installing & Setting up the Symfony Framework page to find a replacement that fits you best.

Should the reviews doc updated to advise the reproduction of bugs/PRs using one of the skeletons?

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #11692.

### CODE REVIEW

File: contributing/community/reviews.rst [hunk 1]
```diff
@@ -134,9 +134,9 @@ Pick a pull request from the `PRs in need of review`_ and follow these steps:
 #. **Reproduce the Problem**
 
    Read the issue that the pull request is supposed to fix. Reproduce the
-   problem on a clean `Symfony Standard Edition`_ project and try to understand
-   why it exists. If the linked issue already contains such a project, install
-   it and run it on your system.
+   problem on a clean project created with the Symfony skeleton (or website
```
Comment (boite) [inline contributing/community/reviews.rst]:
```suggestion
   problem on a clean project created with the `Symfony skeleton`_  or `Symfony
```

File: contributing/community/reviews.rst [hunk 2]
```diff
@@ -134,9 +134,9 @@ Pick a pull request from the `PRs in need of review`_ and follow these steps:
...
+   skeleton) and try to understand why it exists. If the linked issue already
```
Comment (boite) [inline contributing/community/reviews.rst]:
```suggestion
   website skeleton`_ and try to understand why it exists. If the linked issue already
```

**Review by boite — CHANGES_REQUESTED** [review]
:+1: but change to links a couple more mentions of the skeletons

**Comment by javiereguiluz** [comment]
We usually only show the link the first time something is included in the article ... but these "skeletons" are still not well-known by lots of people, so let's include the links always. Thanks.

File: contributing/community/reviews.rst [hunk 3]
```diff
@@ -58,16 +58,16 @@ The steps for the review are:
...
+   created with the `Symfony skeleton`_ or the `Symfony website skeleton`_
+   that reproduces the bug. If it doesn't, the report should at least contain
+   enough information and code samples to reproduce the bug.
 
 #. **Reproduce the Bug**
 
    Download the reproduction project and test whether the bug can be reproduced
    on your system. If the reporter did not provide a reproduction project,
-   create one by forking_ the `Symfony Standard Edition`_.
+   create one based on one `Symfony skeleton`_.
```
Comment (boite) [inline contributing/community/reviews.rst]:
```suggestion
   create one based on the `Symfony skeleton`_ (or the `Symfony website skeleton`_).
```

**Review by boite — CHANGES_REQUESTED** [review]
Whoops! Sorry for making you do more work; one more change for the sake of consistency.  I've read through the document in full this time. Looks great! Thank you @javiereguiluz.

File: contributing/community/reviews.rst (hunk 3 above)
Comment (javiereguiluz) [reply contributing/community/reviews.rst]: Added too. Thanks.

**Review by boite — APPROVED** [review]

**Pull request merged**